### ISSUE TITLE
tntbuild allows easy source modification

### ISSUE THREAD

**Issue by JohnLouderback** [trueinteractions/tint2#130]
I don't know if tntbuild is just still in development or if this is the intended behavior, but when I build my project, my sources are easily exposed by the "Resources" directory. Modified code in the "Resources" directory is reflected in running the app. Tint v1 allowed source files to be bundled into the executable. Is this something that is/can be on the road map for Tint's package building? It would be very preferable to have the source files be less accessible.

**Comment by trevorlinton** [comment]
@JohnLouderback Yes, its currently on our roadmap.  We can include the resources as part of the executable and plan to.  I've just been debating including it with the commercial version (which is also almost ready) or the free one. 

I guess the argument is if its a core need for apps to obscure source code that it should be in the free version we can include it. Keep in mind, even in Tint v1, obscuring source code into the executable doesn't prevent modifying the source, it just makes it more difficult. The only true way of keeping your source from being modified is through code signing (which the commercial and free version both support).

I'll leave it up to you if you feel it should be in the OS/free version or commercial, either way your qualify for the commercial version once its ready.

**Comment by trevorlinton** [comment]
@JohnLouderback  Also, I added you to the contributor list on Tint2, feel free to push up any changes you feel like make sense to the API.

**Comment by JohnLouderback** [comment]
Hey Trevor,

Thanks for adding me to the contributors list! I'll be sure to help out how I can.

As for the packing feature to be pro only, that's seems like a good compromise to me. The reason I think the packaging feature is good as a pro feature is that it still allows developers to use the SDK in its entirety and then can buy a pro license when they're ready to publish. This allows people to adopt Tint more easily while still having the confidence to fund the project. Packaging is certainly among the key features of Tint that set it apart from the competition and I think would thereby be a driving factor in sales. That's my two cents, anyways.

Also, will the same issues with packaging node modules exist that existed in Tint v1?

Speaking of licenses (and I haven't been too eager to ask), correct me if I'm wrong, but am I coming up on needing to renew my license? I had bought the Profession License previously.

Also, I don't know if you'd be interested in contributions, but is there a repository for the Tint Docs? I see a few features undocumented (which is probably intentional), but I also figured that if I make a push, or a pull request for a feature, it'd be nice to document it as well.
